docs/products/kafka/best-practices.rst

```diff
@@ -0,0 +1,75 @@
+Best practices
+==============
+
+We recommend to follow these best practices to ensure that your Apache Kafka service is fast and reliable.
+
+Apache Kafka
+-------------
+
+Check your topic replication factors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Apache Kafka services rely on replication between brokers to preserve data in
+case of the loss of a node. Consider how business critical the data in
+each topic is and make sure that replication is set high enough for it.
+
+You can set the replication factor in `Aiven web console <https://console.aiven.io/>`_ when you create a new topic or edit an existing one.
+
+.. note:: Do not set the replication factor below 2 in order to prevent data loss from unexpected node termination.
+
+Choose a reasonable number of partitions for a topic
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Too few partitions can cause bottlenecks in data processing. In the most
+extreme case, a single partition means that messages are effectively
+processed sequentially. However, too many partitions causes strain on
+the cluster because of an additional overhead. As you cannot reduce the
```

Comment:
Should we also state than ordering is guaranteed only per partition, meaning that you need to understand for which subset of your data you care about the relative ordering and put all that subset in the same partition?